indexing-service/src/main/java/org/apache/druid/indexing/common/task/CompactionTask.java

```diff
@@ -904,6 +928,8 @@ static class ExistingSegmentAnalyzer
     // For processMetricsSpec:
     private final Set<List<AggregatorFactory>> aggregatorFactoryLists = new HashSet<>();
 
+    private long processingTimeMillis = 0;
```

Comment:
removed. I had thought perhaps a metric for all the time spent in this class would be interesting, but decided against implementing it as it would have resulted in a few new metrics.